docs/sdk/wallets/minting.md

```diff
@@ -1,158 +1,13 @@
 # Minting a PKP
-This page will walk you through the process of creating PKPs using the [V3 SDK](../../migration/overview), including adding [permitted scopes](../wallets/auth-methods#auth-method-scopes), which are now required in order to create [session signatures](../authentication/session-sigs/intro). 
 
-## Mint via Contracts
+A PKP is a public/private key-pair generated by the Lit Network using Distributed Key Generation (DKG), meaning no one node ever has access to the entire private key. Instead, the private key is stored in shares across the network, where each node holds a single share. In other words, each PKP is functionally a wallet, where the private key lives across the Lit Network.
 
-You can mint a PKP NFT from the PKP contract on Chronicle - Lit's custom EVM rollup testnet - using the [Lit explorer](https://explorer.litprotocol.com/mint-pkp), the Lit relayer (sign up for an API key [here](https://forms.gle/RNZYtGYTY9BcD9MEA)) or the contracts directly using the [contracts-sdk](https://js-sdk.litprotocol.com/modules/contracts_sdk_src.html). 
+A PKP is represented as an ERC-721 NFT, and the owner of the NFT becomes the designated “controller” of the Programmable Key Pair. The controller has the ability to assign additional signing logic and [authentication](../authentication/overview) mechanisms to their PKP using Lit Actions.
 
-The NFT represents root ownership of the PKP. The NFT owner can grant other users (via a wallet address) or grant Lit Actions the ability to use the PKP to sign and decrypt data. They also have the ability to assign additional authentication methods, described at the bottom of the page.
+This section will walk you through the process of creating PKPs using the [V3 SDK](../../migration/overview), including adding [permitted scopes](../wallets/auth-methods#auth-method-scopes), which are now required in order to create [session signatures](../authentication/session-sigs/intro). 
```

Comment:
this feels like we are trying to fit too much info into one sentence. I don't think we need to include the permitted scope and session sigs info here but can introduce these concepts later

Reply:
Agreed @a1ttech :)